department-of-veterans-affairs/vets-website
# Fix for search pagination state not updating

## Description
This fixes a bug where state was not updating on the search app from the new searchDropdown component.  It also removes all fallback code that was in place from when this 

## Original issue(s)
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11282


## Testing done
Tested on Local,  steps to QA:
1. On your local vets-website repo, go to the following file: src/platform/utilities/api/index.js
2. Change line 57 from: const baseUrl = `${environment.API_URL}/${apiVersion}`; to const baseUrl = `https://api.va.gov/${apiVersion}`;
3. Search for something on VA.gov
4. Search for something different, using the search input on the results page
5. Observe that results render as expected

## Screenshots
<img width="1240" alt="Screen Shot 2022-10-25 at 9 26 24 AM" src="https://user-images.githubusercontent.com/61624970/197786619-c84e09b8-1768-4d90-8605-fbfd346534e3.png">
<img width="1240" alt="Screen Shot 2022-10-25 at 9 26 39 AM" src="https://user-images.githubusercontent.com/61624970/197786622-de1325cf-8988-48b6-8e20-8319bbd4d1ca.png">
<img width="1240" alt="Screen Shot 2022-10-25 at 9 26 48 AM" src="https://user-images.githubusercontent.com/61624970/197786624-ddf35ebe-2430-4451-8392-0fc2f0561805.png">


## Acceptance criteria
- [ ] Update SearchDropdownComponent to change userInput state changes
- [ ] Verify that the pagination bug no longer occurs
- [ ] Remove old fallback code if it is no longer required

## Definition of done
